Update Service Account descriptions (#13)
rackerlabs/docs-gcp: docs/access-and-permissions/index.rst

access unless necessary, and then grant it at the most granular level
possible is necessary to ensure unintentional access is not granted.

> In order to ensure that your Aviator or Service Blocks projects meet this permissions model,

Reviewer: To ensure that your ...

Author: ```suggestion
To ensure that your Aviator or Service Blocks projects meet this permissions model,
```

> Rackspace may periodically audit the permissions being passed to the project

Reviewer: Rackspace might periodically ...

Author: ```suggestion
Rackspace might periodically audit the permissions being passed to the project
```

> and require adjustments to utilize the least permissive model.

Reviewer: ... adjustments to use the least ...

Author: ```suggestion
and require adjustments to use the least permissive model.
```

Service Accounts
^^^^^^^^^^^^^^^^

> Rackspace will add a service account with the Project Owner role to each of

Reviewer: Rackspace adds a service ...

Author: ```suggestion
Rackspace adds a service account with the Project Owner role to each of
```

your GCP projects that we manage: ``automation@rackspace-mgcp.iam.gserviceaccount.com``.

> Additionally, we will grant these service accounts access with the following roles to enable support tooling for all Aviator and Service Blocks projects:

Reviewer: Additionally, we grant these services ...

Author: ```suggestion
Additionally, we grant these service accounts access with the following roles to enable support tooling for all Aviator and Service Blocks projects:
```

- ``resource-observer@rackspace-mgcp.iam.gserviceaccount.com``

  The Resource Observer collects project metadata for support inventory

  + Viewer

- ``smart-tickets@rackspace-mgcp.iam.gserviceaccount.com``

  Smart Tickets works with Watchman to provide automated diagnostics and additional context for monitoring alerts that are turned to tickets for Rackers to address

  + Viewer
  + IAP-secured Tunnel User
  + Compute Instance Admin
  + Compute Security Admin

- ``mgcp-operations@rackspace-mgcp.iam.gserviceaccount.com``

  MGCP Operations facilitates integration of Operations monitoring (formerly Stackdriver) with Watchman

  + Viewer
  + Monitoring Admin

Do not remove these accounts or alter their permissions in any way without first consulting with your
:ref:`support team <support>`.

> We will also temporarily add accounts from the

Reviewer: We also temporarily add ...

Author: ```suggestion
We also temporarily add accounts from the
```

> gcp.rackspace.com domain as Rackers and automation need access to your

Reviewer: **gcp.rackspace.com** domain ...
... and automations need access ...

Author: ```suggestion
gcp.rackspace.com domain as Rackers and automations need access to your
```
